### PR TITLE
Adding metadata

### PR DESCRIPTION
**Pull request by Niputi**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -976,6 +976,13 @@
     "@types/history" "*"
     "@types/react" "*"
 
+"@types/react-helmet@^5.0.9":
```
Comment (velddev) [inline yarn.lock]: mark yarn.lock as binary please.

Comment (Niputi) [reply yarn.lock]: wha how to do that?

Comment (velddev) [reply yarn.lock]: https://stackoverflow.com/questions/11162267/how-do-i-make-git-treat-a-file-as-binary

Comment (Niputi) [reply yarn.lock]: done

**Pull request merged**